### PR TITLE
msx2_cart.xml: Added 6 items, 5 working.

### PR DESCRIPTION
**Pull request by wilbertpol**
machine/mb87030.cpp: Fix some bit definitions for the SCMD register. [pmackinlay]

msx2_cart.xml: Added 6 items, 5 working. [Wilbert Pol]

New working software list items
-------------------------------
MEGA-SCSI (1024KB) [file-hunter, Wilbert Pol]
MEGA-SCSI (512KB) [file-hunter, Wilbert Pol]
MEGA-SCSI (256KB) [file-hunter, Wilbert Pol]
MEGA-SCSI (128KB) [file-hunter, Wilbert Pol]
DSKPRO (v11.6) [file-hunter]

New NOT_WORKING software list additions
------------------------------------------
Video Digitizer (Japan) [file-hunter]

### CODE REVIEW

File: hash/msx2_cart.xml [hunk 1]
```diff
@@ -3582,6 +3582,26 @@ LZ93A13 (32 pin) - 8KB banks
 		</part>
 	</software>
 
+	<software name="hbiv1" supported="no">
+		<description>HBI-V1 Video Digitizer (Japan)</description>
+		<year>1989</year>
+		<publisher>Sony</publisher>
+		<notes>Digitizer cartridge not emulated.</notes>
+		<info name="alt_title" value="ビデオデジタイザー"/>
```
Comment (cuavas) [inline hash/msx2_cart.xml:3590]: If you’re including the model number in the title, it should probably be in both titles.  I’d probably make it “Video Digitizer HBI-V1 (Japan)” and “ビデオデジタイザー HBI-V1” as the box and manual have “Video Digitizer” most prominent.

File: src/devices/bus/msx/cart/scsi.cpp [hunk 2]
```diff
@@ -5,25 +5,29 @@
 
 #include "bus/nscsi/devices.h"
 #include "machine/nscsi_bus.h"
+#include "machine/mb87030.h"
 #include "machine/wd33c9x.h"
```
Comment (cuavas) [inline src/devices/bus/msx/cart/scsi.cpp:9]: `#include` statements not sorted.

File: src/devices/machine/mb87030.h [hunk 3]
```diff
@@ -74,9 +74,9 @@ class mb87030_device : public nscsi_device, public nscsi_slot_card_interface
 	constexpr static uint8_t SCTL_RESET_AND_DISABLE = 0x80;
 
 	constexpr static uint8_t SCMD_TERM_MODE = 0x01;
-	constexpr static uint8_t SCMD_PRG_XFER = 0x02;
-	constexpr static uint8_t SCMD_INTERCEPT_XFER = 0x04;
-	constexpr static uint8_t SCMD_RST_OUT = 0x08;
+	constexpr static uint8_t SCMD_PRG_XFER = 0x04;
+	constexpr static uint8_t SCMD_INTERCEPT_XFER = 0x08;
+	constexpr static uint8_t SCMD_RST_OUT = 0x10;
```
Comment (cuavas) [inline src/devices/machine/mb87030.h:79]: Have you checked that this doesn’t break the HP9000/300 series machines that use this device?

Comment (wilbertpol) [reply src/devices/machine/mb87030.h:79]: Before and after this change the hp9k drivers fail to boot the os from the hdd image in the software list resulting in a 'Root device busy' message and halting the boot process.
Similarly trying to install the os on a clean harddisk image results in an error when creating the filesystem.

File: hash/msx2_cart.xml (hunk 1 above)
Comment (wilbertpol) [reply hash/msx2_cart.xml]: I have removed the HBI-V1 part from the description to match the Japanese name on the box and manual.

File: src/devices/bus/msx/cart/scsi.cpp (hunk 2 above)
Comment (wilbertpol) [reply src/devices/bus/msx/cart/scsi.cpp]: Sorted the includes

**Pull request merged**